src/customHooks/useForm.js

```diff
@@ -0,0 +1,15 @@
+import { useState } from "react";
+
+export function useForm(initialState) {
```

Comment:
add functionality to custom set the value. like 
`setValue('customer', {name: 'abc', email: 'asd@a.com'})`

usage in createInvoice page

`setValue('items', (currentItems)=> ([...currentItems, newItem]))`

second param can be function or value

Reply:
@rparsh Can you please help me with this one. I tired making it. I am confused how to do it.

Reply:
```js

const updateValue = (key, val) => {
    if(typeof val === 'function') {
        setValues((ex)=>({...ex, [key]: val(ex[key])}))
    } else {
        setValues((ex)=>({...ex, [key]: val}))
    }
}
```

Reply:
@smilegupta